starport_template/lib/pages/assets_portfolio_page.dart

```diff
@@ -102,6 +105,22 @@ class _AssetsPortfolioPageState extends State<AssetsPortfolioPage> {
   // }
 
   Future _fetchWalletBalances() async {
-    await StarportApp.walletsStore.getBalances(widget.walletInfo.address);
+    await StarportApp.walletsStore.getBalances(selectedWallet.publicAddress);
+  }
+
+  Future<void> _onDropDownTapped() async {
+    final wallet = await showModalBottomSheet(
+      context: context,
+      isScrollControlled: true,
+      enableDrag: true,
+      shape: RoundedRectangleBorder(borderRadius: CosmosTheme.of(context).borderRadiusM),
+      builder: (context) =>
+          SizedBox(height: MediaQuery.of(context).size.height / 1.06, child: const WalletsListSheet()),
```

Comment:
This `1.06` is total height of the screen divided by the height of the opened bottom sheet given in the design n Figma. Which was `812/764`. I think we need to store it somewhere so that we can use it in all bottom sheets.